packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -169,15 +221,58 @@ export class OpcFeedback extends LitElement {
       </p>
     </dialog>
     
+    <!-- Initial Modal -->
+    <dialog id="initial-dialog" class="op-feedback__panel pf-u-mr-0 pf-u-p-0" .open="${this._openInitialModal}">
+      <header class="op-feedback__header">
+        <h3 class="pf-u-font-weight-normal pf-u-font-size-md pf-u-m-0">${this.template.dialogTitle}</h3>
+      </header>
+      <main id="op-feedback__panel-body">
+      <ul class="op-feedback__options pf-u-m-0 pf-u-p-0">
+        <li>
+          <button type="button" autofocus="true" data-feedback-type="bug" class="op-feedback__option-item pf-u-flex-direction-row pf-u-align-items-center pf-u-w-100 pf-u-display-flex" @click="${e => {
+            this.toggle();
+            this._setModalState(false, false, true, false);
+          }}">
+            <ion-icon name="bug-outline" class="op-feedback__option-icon pf-m-text-align-left"></ion-icon>
+            ${this.template.bugReportTitle}
+          </button>
+        </li>
+        <li>
+          <button type="button" data-feedback-type="feedback" class="op-feedback__option-item pf-u-flex-direction-row pf-u-align-items-center pf-u-w-100 pf-u-display-flex" @click="${e => {
+            this.toggle();
+            this._setModalState(false, true, false, false);
+          }}">
+            <ion-icon name="chatbox-ellipses-outline" class="op-feedback__option-icon"></ion-icon>
+            ${this.template.feedbackReportTitle}
+          </button>
+        </li>
+        <li>
+          <a href="${this.docs}" data-feedback-type="feedback-list" class="op-feedback__option-item pf-u-flex-direction-row pf-u-align-items-center pf-u-w-100 pf-u-display-flex">
+            <ion-icon name="document-outline" class="op-feedback__option-icon"></ion-icon>
+            ${this.template.documentationTitle}
+            <ion-icon name="open-outline" class="op-feedback__icon-secondary pf-u-ml-xs"></ion-icon>
+          </a>
+        </li>
+        <li>
+          <a href="${this.spa}" data-feedback-type="feedback-list" class="op-feedback__option-item pf-u-flex-direction-row pf-u-align-items-center pf-u-w-100 pf-u-display-flex">
+            <ion-icon name="chatbubbles-outline" class="op-feedback__option-icon"></ion-icon>
+            ${this.template.spaRedirectTitle}
+            <ion-icon name="open-outline" class="op-feedback__icon-secondary pf-u-ml-xs"></ion-icon>
```

Comment:
Replace ion-icons with svgs

Reply:
What's is problem of using it.

It was there since the first version

Reply:
@riginoommen I dont see any references in the README for the the ion-icons CDN and plus the users will have to include it in their SPA's or it won't work which also is just more network requests

Reply:
I removed it